nwbwidgets/ophys.py

```diff
@@ -105,13 +105,7 @@ def update_figure(index=0):
 
 
 def show_df_over_f(df_over_f: DfOverF, neurodata_vis_spec: dict):
-    if len(df_over_f.roi_response_series) == 1:
-        title, data_input = list(df_over_f.roi_response_series.items())[0]
-        return neurodata_vis_spec[RoiResponseSeries](
-            data_input, neurodata_vis_spec, title=title
-        )
-    else:
-        return neurodata_vis_spec[NWBDataInterface](df_over_f, neurodata_vis_spec)
+    return neurodata_vis_spec[NWBDataInterface](df_over_f, neurodata_vis_spec)
```

Comment:
There is a problem here when we make the `neurodata_vis_spec[RoiResponseSeries` an OrderedDict to include the trialized widget as an option (see the behavior in the `view.py` module). 

Because `show_df_over_f` is a callable the function `nwb2widget` expects this output to be a visualization (not an ordered dict). The simplest solution was to roll back the `show_df_over_f` to return the visualization for the whole `Container/DataInterface` and then the visualization  can be properly display as a root of a `lazy_tab`. That is how it is working right now.

The other option was to modify  `nwb2widget` to handle the case when callables return `OrderDict` but that seemed like a larger modification given that `nwb2widget` is such a central function and this might have non-tested side effects in other visualizations. I opted for the simple solution here as that only touches the visualizations concerning the RoiExtractors and DfOverF objects.